### ISSUE TITLE
[Bug]: An avatar image is not displayed in the notification window

### ISSUE THREAD

**Issue by Octopus2** [nextcloud/desktop#4975]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

The following problem has been plaguing me since version 3.x of the desktop client (up to the current version 3.6).

Background: It syncs with two different servers (no problems btw). Two different Nextcloud versions (24.0.5 and 23.0.8) are running on these. These are the user accounts of two administrators.

Issue: A user's avatar image is not displayed everywhere in the desktop client. It appears correctly in the software's settings window, but not in the notification window (see photos). Updating the avatar in the server software doesn't change anything. Surprising, since the desktop software obviously has the data.

I ask for hints which data are still needed to debug the problem more precisely.

### Steps to reproduce

Don´t know...

### Expected behavior

Avatar image is shown...

### Which files are affected by this bug

none

### Operating system

Windows

### Which version of the operating system you are running.

Windows 10

### Package

Other

### Nextcloud Server version

23.0.8 und 24.0.5

### Nextcloud Desktop Client version

3.6.0

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 3.4.2 to 3.4.4)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by Octopus2** [comment]
Make it clearer...

![192256563-8739fc12-8054-4fd8-bcd8-aafd8ef939d6](https://user-images.githubusercontent.com/4060419/193143595-622c8ec4-8cfd-4244-bfb7-eff140df5c64.png)

**Comment by nursoda** [comment]
Most probably related to https://github.com/nextcloud/desktop/issues/2731

**Comment by claucambra** [comment]
Hi @Octopus2 could you provide debug logs? A debug archive can be created in Settings->General->Create Debug Archive

**Comment by Octopus2** [comment]
Switching from "No proxy" to "Use System Proxy" (https://github.com/nextcloud/desktop/issues/4975) doesnt´t change anything.

@claucambra Here is the Debug Archive
[debug-log.zip](https://github.com/nextcloud/desktop/files/9733750/debug-log.zip)

**Comment by andyxheli** [comment]
Also effected by this.

**Comment by nursoda** [comment]
> Here is the Debug Archive

@Octopus2 I suppose you are not aware that your log contains quite sensitive data, such as "Belege Banken/Postbank|Santander|VoBa". You should delete it. However, I'm not sure if/how that is possible in GitHub. As a quick fix, please edit your post, but save the full URL of the uploaded file locally so that you may contact GitHub staff to remove it.

@claucambra The Nextcloud client project should either provide a means to purge sensitive data from the log, or provide means to create a filtered log that only contain events that you need to debug. Even in filtered output sensitive information must be purged or replaced prior to publicly uploading it. At least, don't encourage users to upload unfiltered and unpurged logs. I consider this a security/privacy incident that the Nextcloud security team should handle.

**Comment by andyxheli** [comment]
Yes I agree Is a pain to upload logs too much sensitive data on there.

**Comment by nursoda** [comment]
> Hi @Octopus2 could you provide debug logs?

I'd also provide logs since I think that this is related to https://github.com/nextcloud/desktop/issues/2731. Please tell me how to filter/purge and where to send it to, e.g. to you personally along with your promise to wipe them after looking into them.

In general, especially from a production system containing customer clouds, providing logs isn't an option without thorough cleansing (and manual checks prior to sending it anywhere).
